docs/source/usage.rst

```diff
@@ -483,3 +483,31 @@ Although multiple test order markers create their own parametrization, it can be
     test_multiple_markers.py::test_two_and_four[index=4-bbb] PASSED          [ 87%]
     test_multiple_markers.py::test_two_and_four[index=4-ccc] PASSED          [100%]
     ============================== 8 passed in 0.02s ==============================
+
+
+Ordering test modules
+---------------------
+
+Sometimes you want to order whole test modules instead of single tests.
+This maybe the case if you are testing several steps of a workflow, where each step
+has a number of independent tests in the same test module, but the test modules have
+to be executed in a certain order, because the workflow steps depend on each other.
```

Comment:
Perhaps add, “…depend on each other, as is for example the case with testing a set of API endpoints” 🤔